### PR TITLE
Wicked: Add OVS bridge legacy test case

### PR DESCRIPTION
**Pull request by jlausuch**
- Related ticket: https://progress.opensuse.org/issues/36184
- Verification run: http://fromm.arch.suse.de/tests/5295

### CODE REVIEW

**Comment by jlausuch** [comment]
@asmorodskyi @cfconrad

File: tests/wicked/advanced/ref/t13_ovs_bridge_legacy.pm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
```
Comment (asmorodskyi) [inline tests/wicked/advanced/ref/t13_ovs_bridge_legacy.pm]:
```suggestion
# Copyright © 2019 SUSE LLC
```

File: tests/wicked/advanced/sut/t13_ovs_bridge_legacy.pm [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    $self->get_from_data('wicked/ifcfg/ovsbr0', $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifup');
+    record_info('INFO', script_output('ovs-vsctl show'));
+    my $res = $self->get_test_result('br0');
+    die if ($res eq 'FAILED');
+}
+
+sub test_flags {
+    return {always_rollback => 1};
```
Comment (asmorodskyi) [inline tests/wicked/advanced/sut/t13_ovs_bridge_legacy.pm]: let's do this when we will got t14

File: tests/wicked/before_test.pm [hunk 3]
```diff
@@ -59,6 +59,9 @@ sub run {
         assert_script_run("rcwickedd restart");
         record_info('INFO', 'Setup OpenVPN');
         zypper_call('--quiet in openvpn', timeout => 200);
+        record_info('INFO', 'Setup OVS');
+        zypper_call('--quiet in openvswitch', timeout => 200);
```
Comment (asmorodskyi) [inline tests/wicked/before_test.pm]: startandstop suite actually don't need that , please wrap it to additional if

Comment (asmorodskyi) [reply tests/wicked/before_test.pm]: whole section of course

File: tests/wicked/advanced/sut/t13_ovs_bridge_legacy.pm (hunk 2 above)
Comment (cfconrad) [reply tests/wicked/advanced/sut/t13_ovs_bridge_legacy.pm]: What about of moving it to wickedbasetest, so only a test which do not wonna have that feature is needed to overwrite this function?

Comment (asmorodskyi) [reply tests/wicked/advanced/sut/t13_ovs_bridge_legacy.pm]: but it can be done in separate PR

Comment (cfconrad) [reply tests/wicked/advanced/sut/t13_ovs_bridge_legacy.pm]: agree

File: tests/wicked/before_test.pm (hunk 3 above)
Comment (jlausuch) [reply tests/wicked/before_test.pm]: done

File: tests/wicked/advanced/sut/t13_ovs_bridge_legacy.pm (hunk 2 above)
Comment (jlausuch) [reply tests/wicked/advanced/sut/t13_ovs_bridge_legacy.pm]: ok

File: tests/wicked/advanced/ref/t13_ovs_bridge_legacy.pm (hunk 1 above)
Comment (jlausuch) [reply tests/wicked/advanced/ref/t13_ovs_bridge_legacy.pm]: ok

**Pull request merged**